pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -17,13 +17,14 @@
 
 from pytorch_lightning import loops  # import as loops to avoid circular imports
 from pytorch_lightning.loops.batch import TrainingBatchLoop
-from pytorch_lightning.loops.optimization.closure import OutputResult
-from pytorch_lightning.loops.utilities import _prepare_dataloader_iter
+from pytorch_lightning.loops.optimization.closure import ClosureResult, OutputResult
```

Comment:
this looks like a bad rebase which is causing the tests to ail

Reply:
yep I made a mistake when merging master in